settings.js

```diff
@@ -1,5 +1,7 @@
 'use strict'
 
+const { versions = { node: '' }, stdout } = require('./lib/process.js')
```

Comment:
Same comment as other places we could just `const process = require('./lib/process.js');` and mostly leave the rest of the file as it was.